app/Http/Controllers/AmortizationController.php

```diff
@@ -106,7 +106,7 @@ public function recommend(AmmortizationService $service)
             $recommendation = Recommendation::create([
                 "staff_id" => $user->id,
                 "customer_id" => request("customer_id"),
-                "type" => request('type'),
+                "type" => 'recommendation',
```

Comment:
Why are we hard coding this?

Reply:
this is solefy for recommendation service. while the other one is for verification